app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -108,6 +127,15 @@ function AcuantCaptureCanvas({
           },
         },
         onImageCaptureFailure,
+        {
+          text: {
+            NONE: t('doc_auth.info.capture_status_none'),
+            SMALL_DOCUMENT: t('doc_auth.info.capture_status_small_document'),
+            GOOD_DOCUMENT: null,
```

Comment:
this is confusing to me --- so the presence of a null attribute means it was good? Is this like the lack of an error message? What if we made it true?

Reply:
> this is confusing to me --- so the presence of a null attribute means it was good? Is this like the lack of an error message? What if we made it `true`?

By default, Acuant doesn't use a static string label for a "good" document. Instead, they'll trigger a countdown "3...2...1...", which is signified by the `null` value here.

Documentation: https://github.com/Acuant/JavascriptWebSDKV11/tree/master/SimpleHTMLApp#acuantcameraui

Poking around at [the source](https://raw.githubusercontent.com/Acuant/JavascriptWebSDKV11/master/SimpleHTMLApp/webSdk/dist/AcuantJavascriptWebSdk.js), I can see that they do a test of `if (userOptions.text.GOOD_DOCUMENT) { /* ... */ } else { /* ... */ }`, which tells me that it _could_ be provided as a static string if we wanted it to be one.

I don't think we will, but we should at least update the type above from `{null} GOOD_DOCUMENT` to `{string?} GOOD_DOCUMENT`

Reply:
> I don't think we will, but we should at least update the type above from `{null} GOOD_DOCUMENT` to `{string?} GOOD_DOCUMENT`

Aaaand, I neglected to do this before merge 🤦  I'll include it with the follow-up changes mentioned in https://github.com/18F/identity-idp/pull/4143#issuecomment-682886690.